app/results.js

```diff
@@ -2,7 +2,7 @@ import Typography from '@mui/material/Typography'
 import Placeholder from '../component/Placeholder'
```

Comment:
Zpětná vazba k rychlosti s UI. Kdy napíšu 13 semaforových znaků na mobilu, tak po kliknutí na tlačítko variant trvá asi 20s, než se zobrazí první varianty. Vím o tom že mám čekat, ale nepoučený uživatel to nijak nepozná. Zároveň po přesunutí logiky do workeru v tu chvíli UI funguje svižně a to okno se dá v průběhu načítání zavřít.